packages/web-pkg/tests/unit/services/previewService.spec.ts

```diff
@@ -87,30 +87,33 @@ describe('PreviewService', () => {
       })
       expect(preview).toEqual(undefined)
     })
-    it('retries when the server returns a 429 status code', async () => {
-      const supportedMimeTypes = ['image/png']
-      const { previewService, clientService } = getWrapper({
-        supportedMimeTypes,
-        version: '1'
-      })
+    it.each([425, 429])(
+      'retries when the server returns a 425 or 429 status code',
```

Comment:
You can interpolate the status into the string, see e.g. https://github.com/owncloud/web/blob/ac3e2cfb0c1fb92b25be136744fe0b79bb33a24a/packages/web-pkg/tests/unit/utils/urlJoin.spec.ts#L17

This improves readability and debugging of the test runs IMO, because the descriptions are much more specific.

Reply:
```suggestion
      'retries when the server returns a %s status code',
```
should work like this, but please try that before committing it ;-)